cwf/gateway/integ_tests/pipelined.yml

```diff
@@ -137,7 +137,7 @@ li_local_iface: li_port
 li_mirror_all: false
 #li_dst_iface: eth4
 
-redis_enabled: false
+redis_enabled: true
```

Comment:
I'm not sure if we should enable this as it'll force dicts to be redis based which I've disabled in prod due to perf issues

Reply:
@koolzz I'm enabling this only for integration tests. Also i'm concerned regarding the fact that it is disabled in prod. I am assuming redis is always turned on in qos code, I also rely on pipelined to check if redis is reachable. Can we decouple redis_enabled check(redis ping) from the converting rule_id/rule_version to using redis based maps. For now I will add redis ping checks to qos code. Thanks for reviewing this